app/src/main/java/com/theuhooi/uhooipicbook/modules/monsterdetail/dancingmonster/DancingMonsterFragment.kt

```diff
@@ -0,0 +1,58 @@
+package com.theuhooi.uhooipicbook.modules.monsterdetail.dancingmonster
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import androidx.appcompat.app.AppCompatDialogFragment
+import androidx.navigation.fragment.navArgs
+import coil.load
+import com.theuhooi.uhooipicbook.R
+import kotlinx.android.synthetic.main.fragment_dancing_monster.view.*
+
+class DancingMonsterFragment : AppCompatDialogFragment() {
+
+    // region Stored Instance Properties
+
+    private val args: DancingMonsterFragmentArgs by navArgs()
+
+    // endregion
+
+    // region View Life-Cycle Methods
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```

Comment:
ふふ…フラグメントの `onCreateView()` の戻り値の `?` は実は取れるんですよ。
私もこないだ PR で教えてもらいましたｗ

```suggestion
    ): View {
```

Reply:
明示的に`NonNull`だと表明するのもよいですね、ありがとうございます！